### PR TITLE
Added two colours and a board visualisation of the board and changed …

### PR DESCRIPTION
**Pull request by VeraRottger**
…the test file name

### CODE REVIEW

**Review by Joeybalk — APPROVED** [review]
Toppie

File: README.md [hunk 1]
```diff
@@ -39,7 +41,14 @@ Start the application and follow the instructions to play the game.
 ### Example Gameplay
 Computer's Secret Code: G, B, R, Y (unknown to the player).
 
-**Player's Guess 1**:
+Crack the code! You have 12 attempts. 
+
+Guess         | Result
```
Comment (chieldecock) [inline README.md]: | ## | Guess  | Result |

**Review by chieldecock — CHANGES_REQUESTED** [review]
1 comment for the header and maybe add an extra space when the attempt counter hits 10, 11 and 12?

**Review by chieldecock — APPROVED** [review]
Biejoetiefoel

**Pull request merged**